samples/house_price_estimates/README.md

```diff
@@ -0,0 +1,109 @@
+
+### Prepare Data Assets and Storage
+
+Edit the following files to include the relevant details of where your data is stored.  You may use an object store of your choice such as AWS S3, IBM Cloud Object Storage or Ceph.
+
+* house-price-demo-secret.yaml
+* testpii-asset.yaml
+* trainpii-asset.yaml
+
+Register in the Data Catalog (katalog) the training and testing data to be used by the pipeline, noting the catalog ID of each.
+
+```bash
+kubectl apply -f data/house-price-demo-secret.yaml -n kubeflow
+kubectl apply -f data/testpii-asset.yaml -n kubeflow
+kubectl apply -f data/trainpii-asset.yaml -n kubeflow
+```
+
+### Register Governance Policies
+
+Run the following commands to register the data governance policies in the [OPA governance engine being used by Fybrik](https://fybrik.io/v1.0/tasks/using-opa/):
+
+The read policy dictates that columns marked as having personal information should be redacted.
+
+Read Policy:
+
+```bash
+kubectl -n fybrik-system create configmap pii-read-policy --from-file=data/pii-read-policy.rego
+		
+kubectl -n fybrik-system label configmap pii-read-policy openpolicyagent.org/policy=rego
+		
+while [[ $(kubectl get cm pii-read-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+```
+
+The write policy allows all data to be written to any location registered with Fybrik as per the [prequisits](../../get_data_endpoints/README.md).
+
+Write Policy:
+
+```bash
+kubectl -n fybrik-system create configmap allow-write-policy --from-file=data/allow-write-policy.rego
+		
+kubectl -n fybrik-system label configmap allow-write-policy openpolicyagent.org/policy=rego
+		
+while [[ $(kubectl get cm allow-write-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+
+```
+
+### Upload Pipeline
+
+Upload either the pipeline-argo.yaml or pipeline-tekton.yaml file via the Kubeflow Pipeline GUI, depending whether you installed kubeflow pipelines with Argo (default) or with Tekton Pipelines.
+
+See slide 17 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing).
+
+### Run the Pipeline
+
+Create a pipeline run via the Kubeflow Pipeline GUI, providing the following parameters:
+
+* test_dataset_id: testpii-csv
+* train_dataset_id:  trainpii-csv
+
+See slide 19 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing).
+
+### View Pipeline Run Details
+
+Click on each step in the Kubeflow Pipeline GUI and view its log to see the output.  
+
```

Comment:
I got the following errors:
```
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'c104b7e2-6f66-4b4c-aaa7-c0da6672def1', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Content-Type-Options': 'nosniff', 'X-Kubernetes-Pf-Flowschema-Uid': '9295c9c3-d49c-4fbb-8289-9771fab1f995', 'X-Kubernetes-Pf-Prioritylevel-Uid': '782e38ef-0933-4d42-8508-7d4a58cc1b6a', 'Date': 'Thu, 15 Sep 2022 16:21:35 GMT', 'Content-Length': '387'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"fybrikapplications.app.fybrik.io is forbidden: User \"system:serviceaccount:kubeflow:pipeline-runner\" cannot create resource \"fybrikapplications\" in API group \"app.fybrik.io\" in the namespace \"kubeflow\"","reason":"Forbidden","details":{"group":"app.fybrik.io","kind":"fybrikapplications"},"code":403}
```

Should we create relevant RBAC rules in the kubeflow ns?